File: app/src/main/java/com/giphy/browser/common/model/FixedWidth.java
package com.giphy.browser.common.model;

import com.squareup.moshi.Json;

public class FixedWidth {
    @Json(name = "webp")
    public String webp;

    @Json(name = "width")
    public String width;

    @Json(name = "height")
    public String height;
}

File: app/src/main/java/com/giphy/browser/common/model/FixedWidth.kt
package com.giphy.browser.common.model

import com.squareup.moshi.Json

data class FixedWidth(
    @field:Json(name = "webp") val webp: String,
    @field:Json(name = "width") val width: String,
    @field:Json(name = "height") val height: String
)
